tools/src/test/java/org/apache/kafka/tools/consumer/group/ShareGroupCommandTest.java

```diff
@@ -873,6 +1059,10 @@ private Runnable describeGroups(ShareGroupCommand.ShareGroupService service) {
         return () -> Assertions.assertDoesNotThrow(service::describeGroups);
     }
 
+    private Runnable deleteOffsets(ShareGroupCommand.ShareGroupService service) {
+        return () -> Assertions.assertDoesNotThrow(() -> service.deleteOffsets());
```

Comment:
I suggest renaming the `deleteOffsets(String groupId, Set<String> topics)` method in the service, and then you can use a method reference here: `service::deleteOffsets`.